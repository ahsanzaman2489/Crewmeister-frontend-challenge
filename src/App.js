import React, {Suspense} from 'react';
import {BrowserRouter, Route, Switch} from "react-router-dom";
import {DataProvider} from "./context/data.context";
import '@fortawesome/fontawesome-free/css/all.min.css';
import 'bootstrap-css-only/css/bootstrap.min.css';
import 'mdbreact/dist/css/mdb.css';
import './App.css';

const HomeRoute = React.lazy(() => import("./routes/Home"));

function App() {
    return (
        <BrowserRouter>
            <Suspense fallback={<div>Loading...</div>}>
                <DataProvider>
                    <Switch>
                        <Route exact path={'/'} component={HomeRoute}/>
                    </Switch>
                </DataProvider>
            </Suspense>
        </BrowserRouter>
    );
}

export default App;
