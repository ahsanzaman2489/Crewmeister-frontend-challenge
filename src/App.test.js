import React from 'react';
import {shallow} from 'enzyme';
import App from './App';

it('should render the Title Component correctly', () => {
    let wrapped = shallow(<App/>);
    expect(wrapped).toMatchSnapshot();
});
