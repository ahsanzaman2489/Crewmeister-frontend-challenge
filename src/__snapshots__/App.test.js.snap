// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Title Component correctly 1`] = `ShallowWrapper {}`;
